types/location_muxer.go

```diff
@@ -0,0 +1,134 @@
+package types
```

Comment:
It should not be used, anywhere else but in app, soon(after this commits)and is the only place where LocationMuxer is found which is only used in app as well.